File: demo-ui/src/const/crud/talentuserinfo.js
export const tableOption = {
  "border": true,
  "index": true,
  "indexLabel": "序号",
  "stripe": true,
  "menuAlign": "center",
  "align": "center",
  "searchMenuSpan": 6,
  "column": [
    {
      "type": "input",
      "label": "id",
      "prop": "id",
      "addDisplay": false,
      "editDisabled": true,
      "addDisplay":false,
      "showDisplay":false,
      "hide":true
    },

    {
      "type": "input",
      "label": "抖音appid",
      "prop": "uid"
    },
    {
      "type": "input",
      "label": "抖音星图id",
      "prop": "uniqueId"
    },
    {
      "type": "input",
      "label": "nickName",
      "prop": "nickName"
    },
    {
      "type": "input",
      "label": "头像地址",
      "prop": "avatarLink",
      "hide":true
    },
    {
      "type": "input",
      "label": "省",
      "prop": "province"
    },
    {
      "type": "input",
      "label": "市",
      "prop": "city"
    },
  
    {"prop": "gender", "span": 12, "type": "select", "label": "性别", 
    "props": {"label": "label", "value": ""}, "dicData": [{"label": "男", "value": "1"}, {"label": "女", "value": "0"}, {"label": "其他", "value": "2"}], 
    "display": true, "editDisabled": false, "dataType": "string"
    },

    {
      "type": "input",
      "label": "点赞数量",
      "prop": "totalLike"
    },
    {
      "type": "input",
      "label": "平均点赞",
      "prop": "avgLike",
      "sortable":true
    },
    {
      "type": "input",
      "label": "粉丝数量",
      "prop": "fansCount"
    },
    {
      "type": "input",
      "label": "关注数量",
      "prop": "focusCount"
    },
    {
      "type": "input",
      "label": "真粉数量",
      "prop": "realFansCount"
    },
    {
      "type": "input",
      "label": "用户提出的价格",
      "prop": "price",
      "hide":true
    },
    {
      "type": "input",
      "label": "计算的真实价值：",
      "prop": "value",
      "hide":true
    },
    {
      "type": "input",
      "label": "用户领域",
      "prop": "type"
    },
    {
      "type": "input",
      "label": "总播放量",
      "prop": "totalPlayNum"
    },
    {
      "type": "input",
      "label": "平均播放量",
      "prop": "avgPlayNum"
    },
    {
      "type": "input",
      "label": "预期播放量",
      "prop": "prePlayNum"
    },
    {
      "type": "input",
      "label": "单位粉丝贡献播放量",
      "prop": "playNumUnit",
      "hide":true
    },
    {
      "type": "input",
      "label": "互动率",
      "prop": "interactionNum"
    },
    {
      "type": "input",
      "label": "总评论数",
      "prop": "totalCommentNum"
    },
    {
      "type": "input",
      "label": "平均评论数",
      "prop": "avgComment"
    },
    {
      "type": "input",
      "label": "总分享数",
      "prop": "totalShareNum"
    },
    {
      "type": "input",
      "label": "平均分享数",
      "prop": "avgShareNum"
    },
    {
      "type": "input",
      "label": "发表的动态数量",
      "prop": "totalDynaicNum"
    },
    {
      "type": "input",
      "label": "传播力",
      "prop": "communicationValue"
    },
    {
      "type": "input",
      "label": "粉丝特征",
      "prop": "fansFeature",
      "hide":true
    },
    {
      "type": "input",
      "label": "年龄",
      "prop": "age"
    },
    {
      "type": "input",
      "label": "用户描述",
      "prop": "signature",
      "hide":true
    },
    {
      "type": "input",
      "label": "视频数",
      "prop": "videoCount"
    },
    {
      "type": "input",
      "label": "isVerified",
      "prop": "isVerified"
    },
    {
      "type": "input",
      "label": "birthday",
      "prop": "birthday"
    },
    {
      "type": "input",
      "label": "cpm",
      "prop": "cpm"
    },
    {
      "type": "input",
      "label": "xtPrePlayNum",
      "prop": "xtPrePlayNum"
    },
    {
      "type": "input",
      "label": "xtCpm",
      "prop": "xtCpm"
    },
    {
      "type": "input",
      "label": "星图上任务完成数",
      "prop": "xtOrderCompleteNum",
      "hide":true
    },
    {
      "type": "input",
      "label": "扩展字段",
      "prop": "other",
      "hide":true
    },
    {
      "type": "input",
      "label": "平台id",
      "prop": "platformId",
      "hide":true

    },
    {
      "type": "input",
      "label": "喜欢的作品数量",
      "prop": "favoritingCount",
      "hide":true
    },
    {
      "type": "input",
      "label": "爬虫扩展字段：来源",
      "prop": "datafrom",
      "hide":true
    },
    {
      "type": "input",
      "label": "星图id",
      "prop": "oid",
      "hide":true
    },
    {
      "type": "input",
      "label": "平均制作周期",
      "prop": "orderAvgTimeCost"
    },
    {
      "type": "input",
      "label": "星图达人指数",
      "prop": "topScore"
    },
    {
      "type": "input",
      "label": "星图合作指数",
      "prop": "cooperateIndex"
    },
    {
      "type": "input",
      "label": "星图传播指数",
      "prop": "spreadIndex"
    },
    {
      "type": "input",
      "label": "达人风格标签",
      "prop": "userStyle",
      "hide":true
    }
  ]
}

File: demo-biz/src/main/java/com/pig4cloud/pig/demo/entity/TalentUserInfo.java
/*
 *    Copyright (c) 2018-2025, lengleng All rights reserved.
 *
 * Redistribution and use in source and binary forms, with or without
 * modification, are permitted provided that the following conditions are met:
 *
 * Redistributions of source code must retain the above copyright notice,
 * this list of conditions and the following disclaimer.
 * Redistributions in binary form must reproduce the above copyright
 * notice, this list of conditions and the following disclaimer in the
 * documentation and/or other materials provided with the distribution.
 * Neither the name of the pig4cloud.com developer nor the names of its
 * contributors may be used to endorse or promote products derived from
 * this software without specific prior written permission.
 * Author: lengleng (wangiegie@gmail.com)
 */
package com.pig4cloud.pig.demo.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import com.pig4cloud.pig.common.mybatis.base.BaseEntity;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import lombok.EqualsAndHashCode;


/**
 *
 *
 * @author pig code generator
 * @date 2022-05-06 14:33:01
 */
@Data
@TableName("talent_user_info")
@EqualsAndHashCode(callSuper = true)
@Schema(name = "")
public class TalentUserInfo extends BaseEntity {

    /**
     * id
     */
    @TableId(type = IdType.ASSIGN_ID)
    @Schema(name="id")
    private Integer id;

    /**
     * 抖音app里的id
     */
    @Schema(name="抖音app里的id")
    private String uid;

    /**
     * 抖音、星图id
     */
    @Schema(name="抖音、星图id")
    private String uniqueId;

    /**
     * nickName
     */
    @Schema(name="nickName")
    private String nickName;

    /**
     * 头像地址
     */
    @Schema(name="头像地址")
    private String avatarLink;

    /**
     * 省
     */
    @Schema(name="省")
    private String province;

    /**
     * 市
     */
    @Schema(name="市")
    private String city;

    /**
     * 0:女，1：男，2：其他
     */
    @Schema(name="0:女，1：男，2：其他")
    private Integer gender;

    /**
     * 点赞数量
     */
    @Schema(name="点赞数量")
    private Integer totalLike;

    /**
     * 平均点赞
     */
    @Schema(name="平均点赞")
    private Integer avgLike;

    /**
     * 粉丝数量
     */
    @Schema(name="粉丝数量")
    private Integer fansCount;

    /**
     * 关注数量
     */
    @Schema(name="关注数量")
    private Integer focusCount;

    /**
     * 真粉数量
     */
    @Schema(name="真粉数量")
    private Integer realFansCount;

    /**
     * 用户提出的价格：1-20s：1000，20s-60s：2000
     */
    @Schema(name="用户提出的价格：1-20s：1000，20s-60s：2000")
    private String price;

    /**
     * 计算的真实价值：
     */
    @Schema(name="计算的真实价值：")
    private Float value;

    /**
     * 用户领域
     */
    @Schema(name="用户领域")
    private String type;

    /**
     * 总播放量
     */
    @Schema(name="总播放量")
    private Integer totalPlayNum;

    /**
     * 平均播放量
     */
    @Schema(name="平均播放量")
    private Integer avgPlayNum;

    /**
     * 预期播放量
     */
    @Schema(name="预期播放量")
    private Integer prePlayNum;

    /**
     * 单位粉丝贡献播放量
     */
    @Schema(name="单位粉丝贡献播放量")
    private Float playNumUnit;

    /**
     * 互动率
     */
    @Schema(name="互动率")
    private Float interactionNum;

    /**
     * 总评论数
     */
    @Schema(name="总评论数")
    private Integer totalCommentNum;

    /**
     * 平均评论数
     */
    @Schema(name="平均评论数")
    private Integer avgComment;

    /**
     * 总分享数
     */
    @Schema(name="总分享数")
    private Integer totalShareNum;

    /**
     * 平均分享数
     */
    @Schema(name="平均分享数")
    private Integer avgShareNum;

    /**
     * 发表的动态数量
     */
    @Schema(name="发表的动态数量")
    private Integer totalDynaicNum;

    /**
     * 传播力
     */
    @Schema(name="传播力")
    private Float communicationValue;

    /**
     * 粉丝特征
     */
    @Schema(name="粉丝特征")
    private String fansFeature;

    /**
     * 年龄
     */
    @Schema(name="年龄")
    private Integer age;

    /**
     * 用户描述
     */
    @Schema(name="用户描述")
    private String signature;

    /**
     * 视频数
     */
    @Schema(name="视频数")
    private Integer videoCount;

    /**
     * isVerified
     */
    @Schema(name="isVerified")
    private Integer isVerified;

    /**
     * birthday
     */
    @Schema(name="birthday")
    private String birthday;

    /**
     * cpm
     */
    @Schema(name="cpm")
    private Float cpm;

    /**
     * xtPrePlayNum
     */
    @Schema(name="xtPrePlayNum")
    private Integer xtPrePlayNum;

    /**
     * xtCpm
     */
    @Schema(name="xtCpm")
    private Float xtCpm;

    /**
     * 星图上任务完成数
     */
    @Schema(name="星图上任务完成数")
    private Integer xtOrderCompleteNum;

    /**
     * 扩展字段
     */
    @Schema(name="扩展字段")
    private String other;

    /**
     * 平台id
     */
    @Schema(name="平台id")
    private Integer platformId;

    /**
     * 喜欢的作品数量
     */
    @Schema(name="喜欢的作品数量")
    private Integer favoritingCount;

    /**
     * 爬虫扩展字段：来源
     */
    @Schema(name="爬虫扩展字段：来源")
    private String datafrom;

    /**
     * 星图id
     */
    @Schema(name="星图id")
    private String oid;

    /**
     * 平均制作周期
     */
    @Schema(name="平均制作周期")
    private Integer orderAvgTimeCost;

    /**
     * 星图达人指数
     */
    @Schema(name="星图达人指数")
    private Integer topScore;

    /**
     * 星图合作指数
     */
    @Schema(name="星图合作指数")
    private Integer cooperateIndex;

    /**
     * 星图传播指数
     */
    @Schema(name="星图传播指数")
    private Integer spreadIndex;

    /**
     * 达人风格标签
     */
    @Schema(name="达人风格标签")
    private String userStyle;


}
